test/contracts/Swap.zol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: CC0
```

Comment:
This is the same contract as in test/contracts/Assign-Shared-Secret.zol

Reply:
Removed it. thanks